backend/app.js

```diff
@@ -0,0 +1,44 @@
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
+var cookieParser = require('cookie-parser');
```

Comment:
Will we be needing this?

Reply:
probably not lines 1 and 4, but we definitely need express and I believe path when we are setting up routes